.github/workflows/go-basic-tests.yaml

```diff
@@ -71,15 +71,15 @@ jobs:
   Environment-Test:
     name: Create cross-platform build
     # needs: [ Setup-Environment ]
-    runs-on: ubuntu-20.04 #${{ matrix.os }}
+    runs-on: ubuntu-latest
     env:
-      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      GITHUB_TOKEN: ${{ secrets.GH_PERSONAL_ACCESS_TOKEN }}
       CGO_ENABLED: ${{ inputs.CGO_ENABLED }}
     # strategy:
     #   matrix:
     #     os: ${{ fromJSON(needs.Setup-Environment.outputs.matrix) }}
     steps:
-      - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # ratchet:actions/checkout@v3
+      - uses: actions/checkout@v4
```

Comment:
Can you pin this GitHub action to a specific commit like it was before?
Just need to run `ratchet pin .github/workflows/go-basic-tests.yaml` to do that.
You can download from here: https://github.com/sethvargo/ratchet

Reply:
i will open a ticket for devops to resolve that. All our kubescape PR's are broken due to that issue